### PR TITLE
#2807 - Cast to int in router for assessment page

### PR DESCRIPTION
**Pull request by katrina-aot-git**
Follow up to #2807 to add parsing the number for the id.

### CODE REVIEW

File: sources/packages/web/src/views/student/NoticeOfAssessment.vue [hunk 1]
```diff
@@ -77,13 +77,13 @@ export default defineComponent({
...
       noaApprovalStatus: AssessmentStatus,
-      assessmentId: number,
+      currentAssessmentID: number,
     ) => {
       viewOnly.value = !(
         applicationStatus === ApplicationStatus.Assessment &&
         noaApprovalStatus === AssessmentStatus.required
       );
-      currentAssessmentId.value = assessmentId;
+      currentAssessmentId.value = currentAssessmentID;
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/student/NoticeOfAssessment.vue:86]: Please adjust the case in the "ID", it should be `currentAssessmentId`.

Comment (andrewsignori-aot) [inline sources/packages/web/src/views/student/NoticeOfAssessment.vue:86]: Missed in the previous PR but the below also need to be adjusted to remove a warning.
```ts
 assessmentDataLoaded: (
      applicationStatus: ApplicationStatus,
      noaApprovalStatus: AssessmentStatus,
      currentAssessmentId: number,
    ) => {
      return (
        !!applicationStatus && !!noaApprovalStatus && !!currentAssessmentId
      );
    },
```
https://github.com/bcgov/SIMS/blob/9c9cbf39c0cb08c2ca3587a62f2c51b87356c0a3/sources/packages/web/src/components/common/NoticeOfAssessmentFormView.vue#L81-L87

Comment (katrina-aot-git) [reply sources/packages/web/src/views/student/NoticeOfAssessment.vue:86]: I've got this now.

Comment (katrina-aot-git) [reply sources/packages/web/src/views/student/NoticeOfAssessment.vue:86]: The reason I kept making different names (like the previous currAssessmentId) was to keep the three variables named distinctly. I'd changed this one to be different from the other assessmentId on the page, and different than currentAssessmentId so there wasn't currentAssessmentId.value = currentAssessmentId. I can change it back to assessmentId as that overlap at least is slightly less confusing.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for making the changes and the fix on the routes, look good 👍

**Pull request merged**